File: test_julia/random42/Julia/operand_generator.jl
using MatrixGenerator

function operand_generator()
    M231::Array{Float64,2} = generate((1500,1300), [Shape.General, Properties.Random(-1, 1)])
    M232::Array{Float64,2} = generate((1300,1500), [Shape.General, Properties.Random(-1, 1)])
    M233::Array{Float64,2} = generate((1600,1300), [Shape.General, Properties.Random(-1, 1)])
    M235::Array{Float64,2} = generate((1600,1500), [Shape.General, Properties.Random(-1, 1)])
    M234::Array{Float64,2} = generate((1500,1600), [Shape.General, Properties.Random(-1, 1)])
    M236::Array{Float64,2} = generate((1500,1600), [Shape.General, Properties.Random(-1, 1)])
    return (M231, M232, M233, M235, M234, M236,)
end

File: test_julia/random42/Matlab/operand_generator.m
function [out] = operand_generator()
    import MatrixGenerator.*;
    out{ 1 } = generate([1500,1300], Shape.General(), Properties.Random([-1, 1]));
    out{ 2 } = generate([1300,1500], Shape.General(), Properties.Random([-1, 1]));
    out{ 3 } = generate([1600,1300], Shape.General(), Properties.Random([-1, 1]));
    out{ 4 } = generate([1600,1500], Shape.General(), Properties.Random([-1, 1]));
    out{ 5 } = generate([1500,1600], Shape.General(), Properties.Random([-1, 1]));
    out{ 6 } = generate([1500,1600], Shape.General(), Properties.Random([-1, 1]));
end